code/game/objects/items/cards_ids.dm

```diff
@@ -250,6 +250,29 @@
 	iff_signal = TGMC_LOYALIST_IFF
 	var/dogtag_taken = FALSE
 
+/obj/item/card/id/dogtag/canStrip(mob/stripper, mob/owner)
+	. = ..()
+	if(!.)
+		return
+	if(dogtag_taken)
+		stripper.balloon_alert(stripper, "Info tag already taken")
+		return FALSE
+	if(owner.stat != DEAD)
+		stripper.balloon_alert(stripper, "[owner] isn't dead yet")
+		return FALSE
+
+/obj/item/card/id/dogtag/special_stripped_behavior(mob/stripper, mob/owner)
+	if(dogtag_taken)
+		return
+	to_chat(stripper, span_notice("You take [owner]'s information tag, leaving the ID tag"))
```

Comment:
balloon?

Reply:
I wanted the extra info which'd be too wordy for just a balloon, but I've added a shorter one on top now.